src/sous-chef/templates/system/menu.html

```diff
@@ -30,10 +30,10 @@ <h4 class="ui header name">
             </div>
         </form>
     </div>
-    <a class="item" href="{% url 'order:list' %}">{% trans 'Orders' %}
+    <a class="item" href="{% url 'order:list' %}?status={{ ORDER_FILTER_DEFAULT_STATUS | urlencode }}&delivery_date={{ ORDER_FILTER_DEFAULT_DATE | date:'SHORT_DATE_FORMAT' | urlencode }}">{% trans 'Orders' %}
```

Comment:
The date filter does not work with the formatting you provide.

`ORDER_FILTER_DEFAULT_DATE | date:'Y-m-d' `